include/vrv/transpose.h

```diff
@@ -0,0 +1,34 @@
+//
+// Name: transpose.h
+// Author: Zach DeCook
+// Created: December 5 2019
+//
+
+#ifndef __VRV_TRANSPOSE_H__
+#define __VRV_TRANSPOSE_H__
+
+#include <iostream>
+
+namespace vrv {
+
+class TPitch;
+
+class TPitch {
+public:
```

Comment:
In this case the member variables are public, so it seems less appropriate for such variables to be prefixed by `m_` (TPitch acts more like a struct rather than a class).   I could make the variables private and access with member functions (that would make `m_` prefix more appropriate).

Reply:
I am fine having them public but I would still prefer to have them prefixed. (I know there are a few cases where this is not the case, such as with Point)

Reply:
Done.